functors_task_1/student/include/impl.h

```diff
@@ -16,16 +46,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
 {
+    return beer == BeerBrand::Corona || beer == BeerBrand::HoeGaarden ? true : false;
```

Comment:
please try to avoid naked true and false in your code, you may directly return result of expression